GenderPayGap.WebUI/Views/Scope/OutOfScopeQuestions.cshtml

```diff
@@ -45,8 +45,11 @@
                             new Conditional
                             {
                                 Html = @<text>
-                                           @(await Html.GovUkLabel(new LabelViewModel {For = nameof(FeedbackViewModel.OtherSourceText), Text = "Please provide some details"}))
-                                           @(await Html.GovUkCharacterCountFor(m => m.WhyOutOfScopeDetails))
+                                           @(await Html.GovUkCharacterCountFor(m => m.WhyOutOfScopeDetails,
+                                               labelOptions: new LabelViewModel
+                                               {
+                                                   Text = "Please provide some details"
+                                               }))

```

Comment:
It looks like `GovUkCharacterCountFor` throws an error if the label is not provided as an option. I didn't catch it earlier because it was a TextArea, which did not throw the same error.

All other charactercounts do have that field, so it was this one case.